docs/reference/errors-and-warnings/NU5128.md

```diff
@@ -0,0 +1,84 @@
+
+### Solution
+
+* Run pack on the project
+
+If possible, use [NuGet's MSBuild pack task](../msbuild-targets.md), as it automatically matches assembly TFMs with dependency groups from the project's target frameworks. Note that your project must use `PackageReference` for its own NuGet dependencies. If your project uses packages.config, you need to use `nuget.exe pack` and a `nuspec` file.
+
+* Manually edited `nuspec` file
+
+Add the reported TFM as an additional Target Framework for which your project compiles for, and add the assemblies to the package. If you are using an SDK style project to multi-target multiple TFMs, NuGet's MSBuild pack targets can automatically add assemblies in the correct `lib/<tfm>/` folder and create dependency groups with the correct TFMs and dependencies. If you are using a non-SDK style project, you will likely need to create an additional project file for the additional TFM, and modify your `nuspec` file to copy the output assemblies in the correct location in the package.
+
+* Add an empty `_._` file
+
+If your package does not contain any assemblies, such as a meta-package, consider adding an empty `_._` file to the `lib/<tfm>/` directories for the TFMs listed in the warning message. For example, if the warning says to add assemblies for the `netstandard2.0` target framework, create an empty `lib/netstandard2.0/_._` file in your package.
+
+* Remove the dependency group
+
+If you use a custom `nuspec` file, remove the dependency group for the reported TFM, leaving only dependency groups for TFMs for which `lib/<tfm>/` or `ref/<tfm>/` files exist.
+
+* Remove all dependencies for packages that are not related to assemblies
+
+If your package does not contain any `lib/` or `ref/` files and is not a meta-package, it likely does not have any dependencies that the package consumer needs. If you are packing with NuGet's MSBuild Pack target, you can set `<SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>` in any `PropertyGroup` in your project file. If you are using a custom `nuspec` file, remove the `<dependencies>` element.
+
+* Other scenarios
+
+This warning was added during NuGet 5.3's development, and first was available in .NET Core SDK 3.0 Preview 9. [NuGet/Home#8583](https://github.com/nuget/home/issues/8583) tracks an issue where the warning was being raised in too many scenarios. You can use the `NoWarn` MSBuild property (add `<NoWarn>$(NoWarn);NU5128</NoWarn>` to any `PropertyGroup` in your project file). If you have multiple projects affected, you can use [`Directory.Build.targets`](/visualstudio/msbuild/customize-your-build) to automatically add `NoWarn` to all projects.
```

Comment:
Is this relative path correct ?
/visualstudio/msbuild/customize-your-build

Reply:
https://review.docs.microsoft.com/en-us/help/contribute/links-how-to?branch=master#links-to-articles-in-other-repos-or-docsets

You can also click the "view" link under preview url in the comment created by the docs bot that impersonates Karan. Click the link in the docs, although it will fail because the target docset doesn't have a branch with the same name as my PR. But, edit the URL to take out `review` from the hostname (or change the branch name to master, as the docs help suggests), and you can see the URL will work when it's live